storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManager.java

```diff
@@ -449,14 +450,13 @@ private void initializeResources() {
                 } catch (Exception e) {
                     log.error("Encountered error while initializing producer/consumer", e);
```

Comment:
Should we mark the initialization as failed when we fail to initialize the producer/consumer? Otherwise, the metadata events won't be propagated.

Reply:
Done !!